### DISCUSSION TITLE
Where can I set that I want to have German subtitles?

### DISCUSSION THREAD

**Discussion by hansnolte** [Tyrrrz/YoutubeDownloader#467] [Q&A]
It seems to work, as you can read here,
https://github.com/Tyrrrz/YoutubeDownloader/issues/387
but where do I set it?

Best regards
Hans

EDIT:
I found another one.
https://github.com/Tyrrrz/YoutubeDownloader/issues/458#issuecomment-2195820445
But that doesn't work for me either.

**Comment by DiddleDiddler** [comment]
#### I am not sure what you mean exactly by "Where can I set that I want to have German subtitles?" but I have some answers for you.

#### YoutubeDownloader should automatically download all the subtitles from the video.

## I have a couple of methods for you to try:

### First method

**YoutubeDownloader** will automatically download all the subtitles from a video **(if there are subtitles available)** and include them in the mp4 file. If you use VLC to view the video, you can open the video in VLC and click on **`Subtitle`**, then **`Sub Track`**. By default, it is set to **`Disable`**, but if there are subtitles available, you can select the desired language.

### Second method

If YoutubeDownloader doesn't meet your requirements, you can try using alternatives. There are online tools like **[downloadyoutubesubtitles](http://www.downloadyoutubesubtitles.com)** or you can use the **[youtube-subtitles-downloader](https://github.com/eliascotto/youtube-subtitles-downloader)** project on GitHub.

I hope this information helps you solve the issue. Let me know if you have any further questions.

> ### EDIT:
> #### You can go to the settings and turn on `Inject subtitles` if it doesn't automaticly put subtitles in you mp4 file.

**Comment by hansnolte** [comment]
Hi CheesymoonBrainstorm,

many thanks for your response.

Sorry, I forgot to delete this thread.

Unfortunately the 3D Youtube Downloader is broken.

Tyrrrz responded to my query here.
https://github.com/Tyrrrz/YoutubeDownloader/issues/458#issuecomment-2232858288


Thank you again for your effort.
Hans
